data/movie/Toy Story (1995)/2010-special-edition-blu-ray/disc01.json

```diff
@@ -1762,6 +1762,11 @@
       "Duration": "0:07:49",
       "Size": 457531392,
       "DisplaySize": "436.3 MB",
+      "Item": {
```

Comment:
You can leave out the edit of the json file and it will be created by the running finalize on the release once the summary file has been edited

Reply:
Interesting, I'll keep that in mind.